client/pom.xml

```diff
@@ -0,0 +1,130 @@
+<project
```

Comment:
I see that you have split it into two projects -> two jars. We are currently discussing the structure of the project and how packaging will be. You probably want to be part of that discussion too. We will be forking this repo soon for a major work on the Java-SDK.

Reply:
Happy to connect with the team, @artursouza ! Where are discussions being held?

Reply:
We just moved to a public issue: https://github.com/dapr/java-sdk/issues/16